src/lib.rs

```diff
@@ -116,7 +116,7 @@ impl IpfsOptions {
             keypair: Keypair::generate_ed25519(),
             mdns: Default::default(),
             bootstrap: Default::default(),
-            kad_protocol: Default::default(),
+            kad_protocol: Some("/ipfs/lan/kad/1.0.0".to_owned()),
```

Comment:
```suggestion
            // default to lan kad for go-ipfs use in tests
            kad_protocol: Some("/ipfs/lan/kad/1.0.0".to_owned()),
```

or something like this? I at least will forget the reason for in about 60 seconds :)